Jellyfin.Server.Implementations/Item/BaseItemRepository.cs

```diff
@@ -1060,7 +1060,7 @@ public static BaseItemDto DeserialiseBaseItem(BaseItemEntity baseItemEntity, ILo
 
         if (dto is null)
         {
-            dto = Activator.CreateInstance(type) as BaseItemDto ?? throw new InvalidOperationException("Cannot deserialise unknown type.");
+            dto = Activator.CreateInstance(type) as BaseItemDto ?? throw new InvalidOperationException("Cannot Deserialize unknown type.");
```

Comment:
```suggestion
            dto = Activator.CreateInstance(type) as BaseItemDto ?? throw new InvalidOperationException("Cannot deserialize unknown type.");
```

Most IDE's have a "preserve capitalization" option when using find & replace 😉